pages/gall/[id]/index.tsx

```diff
@@ -181,7 +181,7 @@ const Gall = ({ species }: Props): JSX.Element => {
                     <Col id="description" className="lead p-3">
                         {selectedSource && selectedSource.description && (
                             <span>
-                                <p className="small">{deserialize(selectedSource.description)}</p>
+                                <p className="small white-space-pre-wrap">{deserialize(selectedSource.description)}</p>
```

Comment:
While we are doing this we should also do it on the host page which shows the same description. One of these days we need to factor out all of the shared code between the gall and host page... maybe once I finish this major rework I am doing now.

Reply:
pushed a commit that covers the description on the host page